tests/datasets/test_sql.py

```diff
@@ -275,34 +275,31 @@ def test_filter_join_ne_array_value(rc, tmp_path, geodb):
     ]
 
 
-@pytest.mark.skip('todo')
 def test_filter_multi_column_pk(rc, tmp_path, geodb):
     create_tabular_manifest(tmp_path / 'manifest.csv', striptable('''
-    id | d | r | b | m | property | source      | prepare            | type   | ref           | level | access | uri | title   | description
-       | datasets/gov/example     |             |                    |        |               |       |        |     | Example |
-       |   | data                 |             |                    | sql    |               |       |        |     | Data    |
-       |   |   |                  |             |                    |        |               |       |        |     |         |
-       |   |   |   | country      | salis       |                    |        | id,code       |       |        |     | Country |
-       |   |   |   |   | id       | id          |                    | string |               | 3     | open   |     | Code    |
-       |   |   |   |   | code     | kodas       |                    | string |               | 3     | open   |     | Code    |
-       |   |   |   |   | name     | pavadinimas |                    | string |               | 3     | open   |     | Name    |
-       |   |   |                  |             |                    |        |               |       |        |     |         |
-       |   |   |   | city         | miestas     | country.code!='ee' |        | name          |       |        |     | City    |
-       |   |   |   |   | name     | pavadinimas |                    | string |               | 3     | open   |     | Name    |
-       |   |   |   |   | country  | salis       |                    | ref    | country[code] | 4     | open   |     | Country |
+    id | d | r | b | m | property | source      | prepare            | type    | ref            | level | access | uri | title   | description
+       | datasets/gov/example     |             |                    |         |                |       |        |     | Example |
+       |   | data                 |             |                    | sql     |                |       |        |     | Data    |
+       |   |   |                  |             |                    |         |                |       |        |     |         |
+       |   |   |   | Country1     | salis       |                    |         | id,code        |       |        |     | Country |
```

Comment:
Change `datasets/gov/example` to `keymap` (or something like that) instead of changing model name.